dcos/cosmospackage.py

```diff
@@ -562,8 +583,11 @@ def _get_header(request_type, version):
     :rtype: str
     """
 
-    return ("application/vnd.dcos.package.{}+json;"
-            "charset=utf-8;version={}").format(request_type, version)
+    if request_type == 'add-response':
+        return 'application/vnd.dcos.universe.package+json;charset=utf-8;version=v3'
```

Comment:
Yes, this is a workaround for the current version. @cruhland has already told me this will change.